firmware/Cargo.toml

```
@@ -103,3 +126,7 @@ mpu6050-dmp = { git = "https://github.com/TheButlah/mpu6050-dmp-rs", rev = "7712
 
 defmt = { git = "https://github.com/TheButlah/defmt", rev = "b5d5e316ea56250dcb25b3549bcf4bfdb1687796" }
 defmt-rtt = { git = "https://github.com/TheButlah/defmt", rev = "b5d5e316ea56250dcb25b3549bcf4bfdb1687796" }
+
+[profile.dev.package.esp-wifi]
+# https://github.com/esp-rs/esp-wifi#important
+opt-level = 3
```

Linking to readme like that is not as good as linking to the actual issue.
```suggestion
opt-level = 3 # https://github.com/esp-rs/esp-wifi/issues/88#issuecomment-1332363697
```

---

it is tho, a readme can be updated. An issue response can still be but it really wont

---

The readme does not provide enough context. The point of the link is for a future individual to answer the question: "is this line still relevant in *current year*". They can see the full detail of the original issue and then figure that out for themselves.